hw/ip/aes/aes.core

```diff
@@ -96,6 +96,7 @@ targets:
       ascentlint:
         ascentlint_options:
           - "-wait_license"
+          - "-stop_on_error"
```

Comment:
is there any way to add this into the common lint hjson?

Reply:
unfortunately not (I have tried...). Fusesoc uses the tool options of the core that contains the top level core to be linted.

Reply:
man that sucks... should we try to make a change to fusesoc or something? (or maybe you've tried that).
It just feels like if something needs to be repeated 50 times...we should find another way to do it.

Reply:
Yeah, it's unfortunate we didn't have time to implement a better solution yet. The issue is at https://github.com/olofk/fusesoc/issues/275.

Reply:
Thanks for linking the issue, Phil.